### PR TITLE
fixed option SC_SYSTEM_OWN_MSEED=OFF for using external libmseed

### PR DESCRIPTION
**Pull request by duperray**
after removing harmful backslashes in FindMSEED.cmake,
it appears compilation failed due to unresolved symbols (from mseed library) for instance for scmaster application which use seiscomp3_core lib which in turn use mseed dependancy.
I simply moved the mseed detection and variable settings from src/system/CMakeLists.txt to src/CMakeLists.txt.
I checked compilation on redhat6 and redhat7 using SC_SYSTEM_OWN_MSEED=ON and OFF and verified using ldd on scmaster that the binary was linking the right mseed library.
Only question left: everything worked perfectly well before this patch when using SC_SYSTEM_OWN_MSEED=ON. I don't know if it make sense because libmseed is a dependancy used not only in the src/system. Perhaps cmake has an internal mecanism automagically making libmseed a dependancy even if linking was only explicit in this only src/system subdirectory?

### CODE REVIEW

**Comment by gempa-jabe** [comment]
Linking to libmseed is requested in https://github.com/SeisComP3/seiscomp3/blob/master/src/trunk/libs/seiscomp3/CMakeLists.txt#L106. Actually it should search for libmseed in src/system and set the global variables which are in turn used in trunk. src/CMakeLists.txt defines the order of subdirs that system is evaluated **before** trunk. So it should work ;)
Before accepting this PR I would like to test as well. Thanks again.

**Comment by duperray** [comment]
ok, perhaps in case the mseed library in not part of the seiscomp3 root tree the global variable is not set properly. I noticed that before the patch submitted when you specify SC_SYSTEM_OWN_MSEED=OFF option, linking of scmaster omit libmseed which conduct to unresolved symbols.
for testing on redhat6 you can use external mseed library from:
https://github.com/obspy/obspy/wiki/Installation-on-Linux-via-RPM-Repository#centos

**Comment by gempa-jabe** [comment]
I tested it and your fix seems to be correct. We have to move the detection one level up otherwise the LIBMSEED_LIBRARY variable is not being updated correctly.

**Comment by duperray** [comment]
ok, next time I will try not to submit 3 pull requests for a very modest contribution ;)

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -1,3 +1,18 @@
+OPTION(SC_SYSTEM_OWN_MSEED "Use included libmseed" ON)
+
+IF (NOT SC_SYSTEM_OWN_MSEED)
+	SET(MSEED_REQUESTED_VERSION 2)
+	FIND_PACKAGE(MSEED REQUIRED)
+	SET(LIBMSEED_INCLUDE_DIR ${MSEED_INCLUDE_DIR})
+	SET(LIBMSEED_LIBRARY ${MSEED_LIBRARY})
+ELSE (NOT SC_SYSTEM_OWN_MSEED)
+	SET(LIBMSEED_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/libs/3rd-party/libmseed CACHE INTERNAL "")
```
Comment (gempa-jabe) [inline src/CMakeLists.txt]: The include dir must refer to ${CMAKE_CURRENT_SOURCE_DIR}/system/libs/3rd-party/libmseed

**Comment by gempa-jabe** [comment]
I applied your patch.

**Pull request closed without merging**